src/pages/settings/household/dependent.svelte

```diff
@@ -6,12 +6,12 @@ import { goto } from '@roxi/routify'
 import { Page } from '@silintl/ui-components'
 
 const onCancel = () => {
-  $goto('../settings')
+  $goto('../household')
```

Comment:
This comment is only semi-related to this PR, but more about spreading tribal knowledge.

We've found it helpful to use root-relative paths in calls to `$goto()`, so as we're adding new calls or editing existing calls, it would be helpful to make them full paths. In this example, that would be 
```suggestion
  $goto('/settings/household')
```